scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -152,11 +152,11 @@ def handle(self, line):
 
         self.process_test(line)
 
-        tc = self.instance.get_case_or_create(self.id)
-        if self.state == "passed":
-            tc.status = "passed"
-        else:
-            tc.status = "failed"
+        for tc in self.instance.testcases:
```

Comment:
if a case is skipped by c ztest_skip, still parse it as Pass? if the whole test is pass?